_data/projects/dotnet-extensions.yml

```
@@ -0,0 +1,9 @@
+---
+name: ".NET Extensions"
+desc: "Extensions for .NET apps, especially cloud services."
+site: https://github.com/dotnet/extensions
+tags:
+- ".net"
+upforgrabs:
+  name: help wanted
+  link: https://github.com/dotnet/extensions/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22
```

:art: a simpler link

```suggestion
  link: https://github.com/dotnet/extensions/labels/help%20wanted
```